extensions/wikia/AdEngine/js/AdEngine2.js

```diff
@@ -192,11 +194,13 @@ define('ext.wikia.adEngine.adEngine', [
 			decorators.push(adDecoratorLegacyParamFormat);
 		}
 
+		log(['run', 'adding adInfo listener'], 'debug', logGroup);
+		adInfoTracker.run();
```

Comment:
Would wrapping it in an `if` with TODO to remove in January be a safer solution? (We can today create a branch removing the wrapper and merge the PR to `dev` after 12/6)